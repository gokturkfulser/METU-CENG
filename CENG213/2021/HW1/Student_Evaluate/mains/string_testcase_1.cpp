#include <iostream>

#include "String.h"
//#include "../String.h"

/*
 * Case 1 : Default constructor; print.
 */
int main() {
    std::cout << "-> Creating a string with default constructor." << std::endl;

    String str;

    std::cout << "-> Printing the string." << std::endl;

    str.print(true);

    return 0;
}
