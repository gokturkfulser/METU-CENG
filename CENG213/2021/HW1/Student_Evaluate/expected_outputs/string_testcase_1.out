-> Creating a string with default constructor.
-> Printing the string.
[]
